custom/covid/management/commands/update_case_index_relationship.py

```diff
@@ -32,6 +32,12 @@ def get_owner_id(case_type):
     return None
 
 
+def get_case_ids(accessor, case_type, inactive_location):
+    if inactive_location:
+        return accessor.get_open_case_ids_in_domain_by_type(case_type, owner_ids=[inactive_location])
```

Comment:
Great point. I wanted an accessor function to filter out both `case_type` and `owner_id`, but the only one is for open cases (no good). I tried adding my own briefly now with no success so I added the check for if a domain is in the inactive domain when the flag is there to the `should_skip`. Although this means it has to iterate through all the contact cases to filter out the ones not in the inactive location, I don't think it will affect the overall time of the script because the bulk of the time is spent with `submit_case_blocks`. Commit here: [d309d82](https://github.com/dimagi/commcare-hq/pull/29185/commits/d309d82f2e62f861761c2fe64d185f4033d918f0)